### ISSUE TITLE
Autodetection not working, likely due to machine having multiple IP addresses

### ISSUE THREAD

**Issue by wjdp** [tpill90/steam-lancache-prefill#260]
I'm running prefill on the same machine as lancache, prefilling only works if I set the DNS resolver of that machine to itself. Without this it does not detect lancache.

I don't want to have this config as the machine is depending on itself for DNS. I went looking for an option to manually point prefill at the correct IP and it seems you did have a DNS override option (seen in #45) but it was removed in lieu of auto-detection.

My machines network setup is:
- 'main IP' `192.168.1.10/24`, this hosts many services and systemd's resolver
- 'lancache IP' `192.168.1.11/24` this only runs the lancache DNS and HTTP servers

I'm guessing autodetection isn't finding the second IP. Any ideas on best way to proceed?

**Comment by tpill90** [comment]
Hi Will, thanks for the report!

I actually have ran into this myself when I was testing out adding another IP to my server,  I just didn't bother fixing since nobody else ran into it yet 😆 

I was able to reproduce this on my server, and I've written a fix for it.  Since I was able to reproduce it, I've confirmed the fix as working on my machine.

Would you mind giving this build another test just to confirm that it resolves your issue?  
https://github.com/tpill90/steam-lancache-prefill/actions/runs/4067257199

**Comment by wjdp** [comment]
Just given this a go, seems to be working brilliantly! Thank you very much for looking into this so quickly.

```
mars ⋊> ~/b/SteamPrefill-1.9.0-linux-x64 dig lancache.steamcontent.com                                          20:17:01

; <<>> DiG 9.18.1-1ubuntu1.3-Ubuntu <<>> lancache.steamcontent.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 44464
;; flags: qr rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;lancache.steamcontent.com.     IN      A

;; ANSWER SECTION:
lancache.steamcontent.com. 6811 IN      CNAME   origin-tier2.steampipe.steamcontent.com.
origin-tier2.steampipe.steamcontent.com. 222 IN CNAME steampipe-origin-tier2.steamcontent.com.
steampipe-origin-tier2.steamcontent.com. 222 IN A 205.196.6.144
steampipe-origin-tier2.steamcontent.com. 222 IN A 205.196.6.174

;; Query time: 32 msec
;; SERVER: 127.0.0.53#53(127.0.0.53) (UDP)
;; WHEN: Wed Feb 01 20:17:08 GMT 2023
;; MSG SIZE  rcvd: 160

mars ⋊> ~/b/SteamPrefill-1.9.0-linux-x64 steam-prefill prefill                                                  20:17:08
[8:17:16 PM] Starting login!
[8:17:17 PM] Logged into Steam
[8:17:17 PM] Steam session initialization complete!

[8:17:19 PM] Starting Metro: Last Light Redux
[8:17:21 PM] Downloading 7.49 GiB
```

**Comment by tpill90** [comment]
Absolutely no problem!  I will include this in the next release.

**Comment by tpill90** [comment]
Released in [v1.9.1](https://github.com/tpill90/steam-lancache-prefill/releases/tag/v1.9.1)
